{"uuid": "4683254d-b204-4380-a1b9-2f88070f3a59", "children": ["f3a3ce7a-86a9-41cf-9266-2cf82e36be2d"], "befores": [{"name": "init_login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=79.0.3945.79)\n\n", "trace": "  File \"D:\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 957, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 783, in call_fixture_func\n    res = next(it)\n  File \"D:\\eclipse\\selenium_learn\\futurn_loan\\test_case\\conftest.py\", line 32, in init_login\n    login_obj.Login(init_driver).login(common_data.username, common_data.pwd)\n  File \"D:\\eclipse\\selenium_learn\\futurn_loan\\pages_object\\login_obj.py\", line 26, in login\n    self.input_text(loc.username_loc, username, \"登录页面输入用户名\")\n  File \"D:\\eclipse\\selenium_learn\\futurn_loan\\common\\basepage.py\", line 77, in input_text\n    ele = self.get_ele(ele_loc, img_doc)\n  File \"D:\\eclipse\\selenium_learn\\futurn_loan\\common\\basepage.py\", line 59, in get_ele\n    ele = self.driver.find_element(*ele_loc)\n  File \"D:\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 978, in find_element\n    'value': value})['value']\n  File \"D:\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1576652576835, "stop": 1576652578307}], "start": 1576652576835, "stop": 1576652578456}